frontend/src/common/FlashMessage.react.tsx

```diff
@@ -16,9 +28,26 @@ export default class FlashMessage extends React.Component<FlashMessageProps, {}>
 
     return (
       <div className={this.props.code}>
-        <p>{this.props.message}</p>
+        <p>
+          {this.props.message} {' '}
+          {this.props.details
+            ? <a
+                className={'FlashMessage-detailsLink' + (this.state.showDetails ? '-displayed' : '-hidden')}
+                href=""
```

Comment:
The `href` parameter should be present (so it behaves like a link) but the value is optional (see [this line](https://github.com/versionpress/versionpress/blob/ac952264da8e22ec43caff77ddce1aea968a5e82/frontend/src/common/FlashMessage.react.tsx#L49))